src/Tribe/Views/V2/Template/Event.php

```diff
@@ -54,8 +56,10 @@ public function __construct( Template_Bootstrap $template_bootstrap ) {
 	 */
 	public function get_path() {
 		$fake_view = View::make( 'reflector' );
-		$path = $fake_view->get_template()->get_template_file( 'index' );
-		return $path;
+		$path      = $fake_view->get_template()->get_template_file( 'default-template' );
+		$override  = V1_Event_Templates::getTemplateHierarchy( 'default-template' );
```

Comment:
This will still use the V1 here, we need to stick to using V2 loading methods even tho will break the compatibility, we can’t use the `getTemplateHierarchy` method.

Reply:
This is the point of the patch, to make the customizations from Avada, Enfold and others work on the single template.